db/migrate/20230907225057_create_theme_settings_migrations.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateThemeSettingsMigrations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :theme_settings_migrations do |t|
+      t.integer :theme_id, null: false
+      t.integer :theme_field_id, null: false
+      t.bigint :version, null: false
```

Comment:
Is there a reason this needs to be of `bigint` type when we're currently limit the version to 4 digits. Even if we do decide to expand the number of digits in the version, I think a `integer` column would suffice?

Reply:
We were considering copying Rails migrations and make the version a timestamp which wouldn't fit in a 32 bit integer, but for usability concerns we decided to go for simple 4-digits incremental version and change it later to timestamps if/when needed.

I've made the column a big integer to make it easier for future us to change migration versioning, but now that I'm thinking more about it, maybe we should start with `integer` and if we end up needing to change the column type, it shouldn't be too tricky after the fact because this table will stay relatively small (therefore quick to rewrite) and not be very busy/critical? What do you think?

Reply:
> maybe we should start with integer and if we end up needing to change the column type, it shouldn't be too tricky after the fact because this table will stay relatively small (therefore quick to rewrite) and not be very busy/critical? What do you think?

I agree with this. Since this table is unlikely to be actively used, a migration to rewrite the column and locking up the table is not a big concern.

Reply:
Ok, I've switched it to `integer`.